codespell_lib/data/dictionary.txt

```diff
@@ -1844,6 +1847,7 @@ brach->branch
 brackeds->brackets
 brackground->background
 bradcast->broadcast
+brakpoint->breakpoint
```

Comment:
You could also add the plural of this (breakpoints).

Reply:
ok